### DISCUSSION TITLE
I'm having trouble getting a specific Node from a Gltf Scene Asset.

### DISCUSSION THREAD

**Discussion by YourRentIsDue** [bevyengine/bevy#4784] [Q&A] [answered]
```Rust
if let Some(map) = assets.get(&map.0) {

        let floor_option= map.named_nodes.get("Floor");
        let floor_option_unwrapped = floor_option.unwrap();
        let asset = assets.get(floor_option_unwrapped);
        println!("{:?}", asset);

    }
```

This what I've got currently. The first variable (floor_option) returns an Option that is Some, more specifically a strong handle, to what I assume is the node I'm trying to get.
Unwrapping it causes no issues.
However, when I try to get the node with assets.get, which is of type Res<Assets<Gltf>> it returns None. I'm not quite sure what I'm doing wrong.

**Comment by DJMcNab** [comment, marked as answer]
My guess would be that this is because your `assets` is of type `Assets<Gltf>`, but to get the `asset` from `floor_option_unwrapped`, you would need `Assets<GltfNode>`.

(To be honest, I was surprised that this compiled, although looking at `Assets::<T>::get::<H>`, it appears that H does not need to be a `Handle<T>`, for reasons unknown)

**Reply by YourRentIsDue** [reply]
As I was writing this question, that's what I suspected as well. Thanks I'll try it out.
Worked like a charm! Thank you!
